### ISSUE TITLE
Ureteral polyp

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4719]
**Preferred term label:**
Ureteral polyp

**Synonyms**
Ureter polyp
Polyp of the ureter

Note -- NCIT:C4530

**Definition (free text, please give PubMed ID)**
A growth protruding from the mucous membrane of the ureter. Ureteral polyps can be attached to the ureter by a broad base or a thin stalk. 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0000069

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peutz Jeghers

**Your nano-attribution (ORCID)**
